nasa-gibs/worldview
# WV-2975: Add 4 TRMM/TMI and AMSRE LPRM soil moisture layers

## Description

Fixes WV-2975

Added  4 TRMM/TMI and AMSRE LPRM soil moisture layers

- LPRM_TMI_Surface_Soil_Moisture_X_Band_Day_Daily
- LPRM_TMI_Surface_Soil_Moisture_X_Band_Night_Daily
- LPRM_AMSRE_Surface_Soil_Moisture_C_Band_Day_Daily
- LPRM_AMSRE_Surface_Soil_Moisture_C_Band_Night_Daily

## How To Test

1. `git checkout wv-2975-add-lprm-tim-amsre-layers`
2. `npm run build && npm start`
3. Go to [this link](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,LPRM_TMI_Surface_Soil_Moisture_X_Band_Day_Daily,LPRM_TMI_Surface_Soil_Moisture_X_Band_Night_Daily,LPRM_AMSRE_Surface_Soil_Moisture_C_Band_Day_Daily,LPRM_AMSRE_Surface_Soil_Moisture_C_Band_Night_Daily,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2003-12-22-T19%3A06%3A24Z) and check to make sure the 4 layers are present and display imagery, layer descriptions and have a data download option.
